gatsby-node.js

```diff
@@ -97,6 +96,22 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
     });
   });
 
+  observabilityPacks.forEach((pack) => {
+    const { name } = pack;
+    const urlNormalizedName = kebabCase(name);
```

Comment:
not super familiar with `kebabCase` but we noticed yesterday there may be special characters in names, so we probably want to slugify the strings to be safe. not sure if `kebabCase` does handles that already.  i think we have a slugify function in docs repo somewhere.

Reply:
found it! here's our [slugify util in docs repo](https://github.com/newrelic/docs-website/blob/develop/scripts/utils/slugify.js)

Reply:
yes! i actually wrote my own and then noticed we had this and wasn't sure how weird it was to have extra characters in the slug. we could filter out anything but alphanumeric and hyphens 🤷‍♀️ or we could just use the ID in the slug and not the title